### PR TITLE
Riga 588/update permissions

### PR DESCRIPTION
**Pull request by pfrilling**
## Summary / Approach
This 
1. adds a new method to revoke workflow permissions from roles
2. Revokes emergency notifications from non-drupal_admin roles.
3. Adds post update to remove role permissions from existing sites
4. Adds post update to unpublish all notifications.

## Testing Instruction(s)
- Confirm only `drupal_admin` has permission to create/edit emergency notification content types.
- Confirm all `emergency_notification` content types are archived/unpublished.

## Metadata
| Question | Answer |
|----------|--------|
| Did you apply meaningful labels to the pull request? |yes
| Documentation reflects changes? |no
| `CHANGELOG` reflects changes? |yes
| Unit/Functional tests cover changes? |no
| Did you perform browser testing? |yes
| Did you provide detail in the summary on how and where to test this branch? |yes
| Risk level |medium
| Relevant links | [RIGA-588](https://thinkoomph.jira.com/browse/RIGA-588)

### CODE REVIEW

File: ecms_base/modules/custom/ecms_workflow/ecms_workflow.module [hunk 1]
```diff
@@ -22,6 +23,22 @@ function ecms_workflow_entity_bundle_create($entity_type_id, $bundle) : void {
     // Call the workflow service to update configuration.
     \Drupal::service('ecms_workflow.bundle_create')
       ->addContentTypeToWorkflow($bundle);
+
```
Comment (montgomeryyoung) [inline ecms_base/modules/custom/ecms_workflow/ecms_workflow.module:26]: Does this revoke the permissions related to the bundle every time an Emergency Notification is created? Or is it that it revokes the permissions from the specific node?

File: ecms_base/modules/custom/ecms_workflow/ecms_workflow.post_update.php [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    ->revokeRolePermissions(
+      EcmsWorkflowBundleCreate::CONTENT_AUTHOR_ROLE,
+      'emergency_notification'
+    );
+}
+
+/**
+ * Revoke permissions from non-drupal_admin roles.
+ */
+function ecms_workflow_post_update_unpublish_all_emergency_notifications(array &$sandbox): void {
```
Comment (montgomeryyoung) [inline ecms_base/modules/custom/ecms_workflow/ecms_workflow.post_update.php:34]: The comment doesn't seem to fit the function

File: ecms_base/modules/custom/ecms_workflow/ecms_workflow.module (hunk 1 above)
Comment (pfrilling) [reply ecms_base/modules/custom/ecms_workflow/ecms_workflow.module:26]: It only revokes permissions when a bundle of that type is created, not for every instance of that node.

File: ecms_base/modules/custom/ecms_workflow/ecms_workflow.post_update.php (hunk 2 above)
Comment (pfrilling) [reply ecms_base/modules/custom/ecms_workflow/ecms_workflow.post_update.php:34]: Fixed.

**Review by montgomeryyoung — APPROVED** [review]
looks good

**Pull request merged**